src/Microsoft.IdentityModel.Tokens/SignatureProvider.cs

```diff
@@ -38,7 +39,7 @@ public abstract class SignatureProvider : IDisposable
         /// <summary>
         /// Maintained by <see cref="CryptoProviderFactory"/>
```

Comment:
Comment is no longer accurate

Reply:
addessed - https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/commit/58e807326fca6a7252db6e8f952a4bf221321810